src/main/java/edu/ucsb/nceas/mdqengine/scheduler/RequestScorerJob.java

```diff
@@ -175,181 +186,205 @@ public void execute(JobExecutionContext context)
             d1Node = DataONE.getMultipartD1Node(session, nodeServiceUrl);
         } catch (MetadigException mpe) {
             mpe.printStackTrace();
-            throw new JobExecutionException(taskName + ": unable to create connection to service URL " + nodeServiceUrl , mpe);
+            throw new JobExecutionException(taskName + ": unable to create connection to service URL " + nodeServiceUrl,
+                    mpe);
         }
 
-        MDQStore store = null;
-
         // Get stored task info from the last task execution
-        try {
-            store = new DatabaseStore();
-        } catch (Exception e) {
-            e.printStackTrace();
-            throw new JobExecutionException("Cannot create store, unable to schedule job", e);
-        }
-
-        if(!store.isAvailable()) {
-            try {
-                store.renew();
-            } catch (MetadigStoreException e) {
-                e.printStackTrace();
-                throw new JobExecutionException("Cannot renew store, unable to schedule job", e);
+        try (DatabaseStore store = new DatabaseStore()) {
+            // TODO: consider removing this block not sure if it's ever hit
```

Comment:
I think you could reduce the nesting here - instead of:

```java
    try (DatabaseStore store = new DatabaseStore()) {
        if (!store.isAvailable()) {
            try {
                store.renew();
            } catch (MetadigStoreException e) {
                e.printStackTrace();
                throw new JobExecutionException(...etc
            }
        }
    }
```

could you do this?

```java
    try (DatabaseStore store = new DatabaseStore()) {
        if (!store.isAvailable()) {
            store.renew();
        }
    } catch (MetadigStoreException e) {
                e.printStackTrace();
                throw new JobExecutionException(...etc
    }
```